code/modules/mob/inventory.dm

```diff
@@ -274,4 +369,47 @@
 	return slot_back
 
 /mob/proc/getBeltSlot()
-	return slot_belt
\ No newline at end of file
+	return slot_belt
+
+
+
+//Inventory.dm is -kind of- an ok place for this I guess
+
+//This is NOT for dismemberment, as the user still technically has 2 "hands"
+//This is for multi-handed mobs, such as a human with a third limb installed
+//This is a very rare proc to call (besides admin fuckery) so
+//any cost it has isn't a worry
+/mob/proc/change_number_of_hands(amt)
+	if(amt < held_items.len)
+		for(var/i in held_items.len to amt step -1)
+			unEquip(held_items[i])
+	held_items.len = amt
+
+	if(hud_used)
+		var/style
+		if(client && client.prefs)
+			style = ui_style2icon(client.prefs.UI_style)
+		hud_used.build_hand_slots(style)
+
+
+/mob/living/carbon/human/change_number_of_hands(amt)
+	var/old_limbs = held_items.len
+	if(amt < old_limbs)
+		for(var/i in hand_organs.len to amt step -1)
+			var/obj/item/bodypart/BP = hand_organs[i]
+			BP.dismember()

```

Comment:
dismember() means a messy dismembering, if you just want to lose a limb then you have drop_limb()